app/channel.go

```diff
@@ -766,8 +766,13 @@ func (a *App) UpdateChannelMemberNotifyProps(data map[string]string, channelId s
 }
 
 func (a *App) DeleteChannel(channel *model.Channel, userId string) *model.AppError {
-	ihc := a.Srv.Store.Webhook().GetIncomingByChannel(channel.Id)
 	ohc := a.Srv.Store.Webhook().GetOutgoingByChannel(channel.Id, -1, -1)
+	ihc := make(chan store.StoreResult, 1)
```

Comment:
nit: could we keep this above the `ohd := ` assignment for "diff consistency". (I'm assuming we'll later change `GetOutgoingByChannel` and it would be nice to keep the order.)